src/js/components/profile-form/spouse.html

```diff
@@ -74,7 +74,7 @@
 							class="btn form-control"
 							ng-class="{'btn-success': ngModel.$valid, 'btn-danger': !ngModel.$valid}"
 							ng-click="addSpouse()">
-						<span class="glyphicon glyphicon-plus" aria-hidden="true"></span> Spouse
+						<span class="glyphicon glyphicon-plus" aria-hidden="true"></span> <span translate>Spouse</span>
```

Comment:
Same here, unless you're trying to prevent the space before Spouse from ending up in the pot. If that's the case, the space would probably be better done with CSS.